scripts/imgtool/keys/aws_kms_ecdsa.py

```diff
@@ -0,0 +1,81 @@
+"""
+RSA Key management
```

Comment:
This comment should probably be a little more specific (and say ECDSA instead of RSA).